packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -479,7 +482,7 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         bytes32 identifier,
         uint256 timestamp,
         bytes memory ancillaryData
-    ) public view override returns (Request memory) {
+    ) public view override nonReentrantView() returns (Request memory) {
```

Comment:
- In the `OptimisticOracle`, `public` `view` functions do not use reentrancy guards. We have not identified a scenario that requires reentrancy protection, but we thought it is worth mentioning because reentrancy guards are used on `public` `view` functions extensively throughout the code base, so this may be an oversight.